src/main/java/org/cyclops/capabilityproxy/tileentity/TileEntityCapabilityProxy.java

```diff
@@ -0,0 +1,37 @@
+package org.cyclops.capabilityproxy.tileentity;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraftforge.common.capabilities.Capability;
+import org.cyclops.capabilityproxy.block.BlockCapabilityProxy;
+import org.cyclops.cyclopscore.tileentity.CyclopsTileEntity;
+
+import java.util.List;
+
+/**
+ * An entity capability proxy.
+ * @author josephcsible
+ */
+public class TileEntityCapabilityProxy extends CyclopsTileEntity {
+
+    public EnumFacing getFacing() {
+        return getWorld().getBlockState(getPos()).getValue(BlockCapabilityProxy.FACING);
+    }
+
+    protected List<Entity> getEntities(Capability<?> capability) {
+        AxisAlignedBB aabb = new AxisAlignedBB(getPos().offset(getFacing()));
+        return getWorld().getEntitiesWithinAABB(Entity.class, aabb, entity -> entity.hasCapability(capability, null));
```

Comment:
All has/getCapability calls here should use `getFacing().getOpposite()` as side. So that the way the block facing works is similar to the other proxies.

Reply:
I thought non-null facing values only made sense for blocks.